tests/basictests/distributed-workloads.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+source $TEST_DIR/common
+
+MY_DIR=$(readlink -f `dirname "${BASH_SOURCE[0]}"`)
+
+RESOURCEDIR="${MY_DIR}/../resources"
+
+source ${MY_DIR}/../../util
+
+os::test::junit::declare_suite_start "$MY_SCRIPT"
+
+function install_codeflare_operator() {
+    header "Installing Codeflare Operator"
+    os::cmd::expect_success "oc apply -f $RESOURCEDIR/codeflare-subscription.yaml"
+
+    # Wait until both pods are ready
+    os::cmd::try_until_text "oc get pods -n openshift-operators | grep "codeflare-operator-controller-manager" | awk '{print \$2}'" "2/2" $odhdefaulttimeout $odhdefaultinterval
+
+    # Ensure that all CRDs are created
+    os::cmd::expect_success_and_text "oc get crd instascales.codeflare.codeflare.dev  | wc -l" "2"
+    os::cmd::expect_success_and_text "oc get crd mcads.codeflare.codeflare.dev | wc -l" "2"
+    os::cmd::expect_success_and_text "oc get crd appwrappers.mcad.ibm.com | wc -l" "2"
+    os::cmd::expect_success_and_text "oc get crd queuejobs.mcad.ibm.com | wc -l" "2"
+    os::cmd::expect_success_and_text "oc get crd schedulingspecs.mcad.ibm.com | wc -l" "2"
+}
+
+function install_distributed_workloads_kfdef(){
+    header: "Installing distributed workloads kfdef"
+    os::cmd::expect_success "oc apply -f ../distributed-workloads-kfdef.yaml"
```

Comment:
Where is this `distributed-workloads-kfdef.yaml` ? did you mean?
```suggestion
    os::cmd::expect_success "oc apply -f $RESOURCEDIR/../../codeflare-stack-kfdef.yaml"
```